contrib/run_function.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+export KUBECONFIG="$(kind get kubeconfig-path)"
+
+kubectl rollout status deploy/gateway -n openfaas
```

Comment:
What does this line do?

Reply:
that should wait for the gateway deployment to register as healthy, i just discovered this command recently as well from my ops team

Reply:
Is it blocking? What is the timeout?

Reply:
Yes it blocks, it returns a nonzero exit code if the deployment misses the progression deadline. I am not sure what that value is, but it is finite

Reply:
We should find out and pass it as a flag if available to self document

Reply:
Ohh. good catch, I should add a check to see if rollout succeeds

Reply:
```
      --timeout=0s: The length of time to wait before ending watch, zero means never. Any other values should contain a
corresponding time unit (e.g. 1s, 2m, 3h).
```

I'd like to see explicit waits done.